.taskcluster.yml

```diff
@@ -55,44 +55,92 @@ tasks:
         owner: bastien@mozilla.com
         source: https://github.com/mozilla/task-boot
 
-    - taskId: {$eval: as_slugid("docker_build")}
-      dependencies:
-        - {$eval: as_slugid("code_checks")}
-      provisionerId: aws-provisioner-v1
-      workerType: releng-svc
-      created: {$fromNow: ''}
-      deadline: {$fromNow: '1 hour'}
-      extra:
-        github:
-          events:
-            - pull_request.opened
-            - pull_request.reopened
-            - pull_request.synchronize
-            - push
-      payload:
-        capabilities:
-          privileged: true
-        maxRunTime: 3600
-        image: python:3-alpine
-        env:
-          # Using a test repo for now
-          TAG: babadie/taskboot
-        command:
-          - sh
-          - -lxce
-          - "apk add git img --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing --quiet &&
-            git clone --quiet ${repository} /src && cd /src && git checkout ${head_rev} -b taskboot &&
-            pip install . &&
-            taskboot --target=/src build --tag=$TAG:latest --write /image.tar Dockerfile"
-        artifacts:
-          public/taskboot/image.tar:
-            expires: {$fromNow: '2 weeks'}
-            path: /image.tar
-            type: file
-      scopes:
-        - docker-worker:capability:privileged
-      metadata:
-        name: TaskBoot Docker build
-        description: Taskcluster boot utilities - build docker image
-        owner: bastien@mozilla.com
-        source: https://github.com/mozilla/task-boot
+    - $if: 'tasks_for == "github-push" && head_branch == "refs/heads/master"'
```

Comment:
Do you want to upload the image for every merged PR? Maybe it'd be better to decide when to release and so only push on tags?

Reply:
Yes, at least for now, it will ease development & trying things.

I plan to add support for release through version tags as on bugbug, but would like to keep `latest` for `master`.
Especially as we can't use the taskcluster `task-image` right now

Reply:
:+1: